hathor/simulator/tx_generator.py

```diff
@@ -84,9 +88,10 @@ def new_tx_step1(self):
         if not self.send_to:
             address = self.manager.wallet.get_unused_address(mark_as_used=False)
         else:
-            address = random.choice(self.send_to)
+            address = self.rng.choice(self.send_to)
 
-        value = random.randint(1, balance.available)
+        value = self.rng.integers(1, balance.available, endpoint=True)
```

Comment:
Is it an `int`?

Reply:
Refactored back to `randint`. But you're right it would be a `numpy.int64`.